module/zcommon/simd_stat.c

```diff
@@ -0,0 +1,167 @@
+#include <sys/zfs_context.h>
```

Comment:
This file needs license header.  It's all new code so BSD clause 2 would be nice.